src/containerapp/azext_containerapp/_params.py

```diff
@@ -395,6 +395,14 @@ def load_arguments(self, _):
     with self.argument_context('containerapp job registry') as c:
         c.argument('identity', help="The managed identity with which to authenticate to the Azure Container Registry (instead of username/password). Use 'system' for a system-defined identity, Use 'system-environment' for an environment level system-defined identity or a resource id for a user-defined environment/containerapp level identity. The managed identity should have been assigned acrpull permissions on the ACR before deployment (use 'az role assignment create --role acrpull ...').")
 
+    with self.argument_context('containerapp job suspend') as c:
+        c.argument('name', name_type, id_part=None, help="The name of the Containerapp job.")
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
+
+    with self.argument_context('containerapp job resume') as c:
+        c.argument('name', name_type, id_part=None, help="The name of the Containerapp job.")
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
```

Comment:
I think you don't need to add these params, because they can inherit the params from[ `az containerapp job`](https://github.com/Azure/azure-cli/blob/9160a9804634c4673128fdf2b2b457220a6f08b9/src/azure-cli/azure/cli/command_modules/containerapp/_params.py#L448)